.vib/matomo/goss/goss.yaml

```diff
@@ -1,36 +1,6 @@
 file:
-  /opt/bitnami/matomo/js:
-    filetype: symlink
-    linked-to: /bitnami/matomo/js
+  /bitnami/matomo:
     exists: true
-  /opt/bitnami/matomo/plugins:
-    filetype: symlink
-    linked-to: /bitnami/matomo/plugins
-    exists: true
-  /bitnami/matomo/config/config.ini.php:
```

Comment:
Should we maintain this test and modify it to make use of the `vars.yaml` file? 

It is true we would only be checking the host and database parameters (which are implicitly checked by Cypress), but checking them here will verify the values passed to the chart are the ones used. Both the hostname and database can fail due to multiple reasons besides this one. wdyt?

Reply:
My opinion is that we should drop it for this kind of checks (db-related, user accounts, ...). We are either already checking them in Cypress or they do not add any value.

GOSS tests are only executed if and only if the deployment was successful. The majority of the apps do fail when the connection to the DB, so that means that if GOSS tests are executed their DB details must be the correct ones.

Reply:
I agree, I don't think this type of test will add enough value to offset the increased test complexity. We may want to document this internally for future reference.